app/javascript/packs/articlePage.jsx

```diff
@@ -23,45 +23,45 @@ top.addSnackbarItem = addSnackbarItem;
 const userDataIntervalID = setInterval(async () => {
```

Comment:
all unrelated formatting changes via prettier (initially I had some code in here but later moved it to the existing `handleFocus` method in `initializeCommentsPage.js.erb`